config/nut/nut.inc

```diff
@@ -333,7 +326,7 @@ EOD;
 EOD;
 
 		$start = <<<EOD
-if [ `pgrep {$driver} | wc -l` != 0 ]; then
+if [ `/bin/pgrep {$driver} | /usr/bin/wc -l` != 0 ]; then
```

Comment:
This single line? Frankly, that pgrep | wc is used consistently in tons of places. I'd rather not touch this code really.